samples/net/sockets/packet/README.rst

```diff
@@ -0,0 +1,28 @@
+.. _packet-socket-sample:
+
+Raw socket sample
```

Comment:
Should this doc title be "Packet socket sample" (to match the label above and the body text below)?

Reply:
Comments are addressed in original PR https://github.com/zephyrproject-rtos/zephyr/pull/12564, thanks.